src/system_info/system_info_display_wayland.cc

```diff
@@ -204,7 +204,7 @@ bool SysInfoDisplay::UpdateBrightness() {
   int val = atoi(str_val);
   free(str_val);
 
-  brightness_ = val / max_val;
```

Comment:
Well, it's needed because otherwise you're doing integer arithmetic (`val` and `max_val` are both `int`).